airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/spec_cache.py

```diff
@@ -1,16 +0,0 @@
-from dagster import asset, AutoMaterializePolicy, FreshnessPolicy
-import pandas as pd
-from metadata_service.spec_cache import list_cached_specs
-from orchestrator.utils.dagster_helpers import OutputDataFrame, output_dataframe
-
-
-GROUP_NAME = "spec_cache"
-
-
-@asset(
-    group_name=GROUP_NAME,
-    auto_materialize_policy=AutoMaterializePolicy.eager(max_materializations_per_minute=30),
-    freshness_policy=FreshnessPolicy(maximum_lag_minutes=1),
-)
-def cached_specs() -> OutputDataFrame:
-    return output_dataframe(pd.DataFrame(list_cached_specs()))
```

Comment:
Is this change related to the "back to 3.9" change or a separate fix for specs?

Reply:
Updated the description to include this :)